From 60ccbd5cc3af93065589ba7e384bad96ba79573b Mon Sep 17 00:00:00 2001
From: Vaibhav Hiremath <hvaibhav@ti.com>
Date: Tue, 29 Nov 2011 18:20:12 +0530
Subject: [PATCH 0632/1092] arm:omap:am33xx: Create an alias for sgx/gfx clock

GFX/SGX driver has specific requriement for clock naming
convention, due to legacy platform support.
So create an alias for the clock "gfx_fclk" --> "sgx_ck".

Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 arch/arm/mach-omap2/board-am335xevm.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/mach-omap2/board-am335xevm.c b/arch/arm/mach-omap2/board-am335xevm.c
index 727d3fe..d77153c 100644
--- a/arch/arm/mach-omap2/board-am335xevm.c
+++ b/arch/arm/mach-omap2/board-am335xevm.c
@@ -1806,6 +1806,9 @@ static void __init am335x_evm_init(void)
 	/* Create an alias for icss clock */
 	if (clk_add_alias("pruss", NULL, "icss_uart_gclk", NULL))
 		pr_err("failed to create an alias: icss_uart_gclk --> pruss\n");
+	/* Create an alias for gfx/sgx clock */
+	if (clk_add_alias("sgx_ck", NULL, "gfx_fclk", NULL))
+		pr_err("failed to create an alias: gfx_fclk --> sgx_ck\n");
 }
 
 static void __init am335x_evm_map_io(void)
-- 
1.7.11.2

